src/ServiceBus/ServiceBus/Cmdlets/CheckNameAvailability/TestAzServiceBusCheckNameAvailability.cs

```diff
@@ -0,0 +1,94 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.ServiceBus.Models;
+using Microsoft.Azure.Commands;
+using System.Collections.Generic;
+using System.Linq;
+using System.Management.Automation;
+using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
+using Microsoft.Azure.Management.ServiceBus.Models;
+
+namespace Microsoft.Azure.Commands.ServiceBus.Commands.Namespace
+{
+    /// <summary>
+    /// 'Test-AzureRmCheckNameAvailability' Cmdlet Check Availability of the NameSpace Name
+    /// </summary>
+    [Cmdlet("Test", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "ServiceBusNameAvailability", DefaultParameterSetName = QueueCheckNameAvailabilityParameterSet), OutputType(typeof(bool))]
+    public class TestAzServiceBusCheckNameAvailability : AzureServiceBusCmdletBase
+    {
+        [Parameter(Mandatory = true, ParameterSetName = QueueCheckNameAvailabilityParameterSet, ValueFromPipelineByPropertyName = true, Position = 0, HelpMessage = "Resource Group Name")]
```

Comment:
@v-Ajnava nit: since there are only two parameter sets for this cmdlet, and this parameter has the same properties in both sets, you can remove this line and on the below line, remove the `ParameterSetName` property. This will make the parameter "global" and apply to all parameter sets for this cmdlet.

Reply:
Resolved